src/Components/SignUp/SignUp.js

```diff
@@ -1,59 +1,274 @@
+import { useState } from "react";
+import { Link, useHistory } from "react-router-dom";
 import styled from "styled-components";
+import { country_arr } from "../../Data/countris";
+import { SiHandshake } from "react-icons/si";
+import CountriesList from "./CountriesList";
+import { postSignup } from "../../Services/TechCommer";
+import {
+  validateInputs,
+  comparePasswords,
+  validEmail,
+} from "../../schemas/userValidations";
 
 export default function SignUp() {
+  const [openContries, setOpenCantries] = useState(false);
```

Comment:
Could you rename this "contries" state to "countries" and "setOpenCantries" state to "setOpenCountries" ?

Reply:
Yes, did!